projects/Mallard/jest.config.js

```diff
@@ -2,6 +2,9 @@ module.exports = {
     preset: 'react-native',
     transformIgnorePatterns: ['/node_modules/(?!(@guardian|react-native))/'],
     moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'],
-    setupFiles: ['./node_modules/react-native-gesture-handler/jestSetup.js'],
+    setupFiles: [
+        './node_modules/react-native-gesture-handler/jestSetup.js',
+        './jest-setup.js',
```

Comment:
Excellent! I'm very happy to see that we can avoid mocking local in multiple places 👍  🎉